include/neml2/models/solid_mechanics/KinematicHardeningYieldFunction.h

```diff
@@ -1,40 +0,0 @@
-// Copyright 2023, UChicago Argonne, LLC
```

Comment:
Why did this go away?  Or is this another weirdo github thing?

Reply:
I was planning to make YieldFunction template on two booleans (which apparently I forgot to do), so that it plays better with the object registration scheme we are using here.